setup.py

```diff
@@ -20,14 +20,15 @@
     'check-manifest>=0.25',
     'coverage>=4.0',
     'httpretty>=0.8.14',
-    'invenio-accounts>=1.0.0',
     'invenio-userprofiles>=1.0.0',
+    'requests-oauthlib>=0.6.2,<1.2.0',
```

Comment:
Does this need to be both in `install_requires` and here?

Reply:
unfortunatelly yes, otherwise it is installing version more than 1.2  from travis